### ISSUE TITLE
Content-Type on DELETE

### ISSUE THREAD

**Issue by awlayton** [OADA/oada-cache#11]
The library seems to require a content type for a DELETE? Is this intended? Seems strange to me.

**Comment by aultac** [comment]
Yes, the way it currently works is a big pain.  As we’ve used it over time, it seems to me that content-type should only be required for initial resource creation: i.e. a PUT/POST if there is no resource there already.  GET, DELETE, and PUT/POST to an existing resource should just use the content-type on the resource already and ignore the content-type header on the server.

We haven’t worked out how to change the content type of a resource which could affect this depending on what we choose to do.

I’m all for removing the requirement for content-type for all cases except resource creation.

Aaron
